src/sentry/static/sentry/less/group-detail.less

```diff
@@ -227,6 +227,10 @@
       background-image: url(../images/logos/context/netcore.svg);
     }
 
+    &.net .context-item-icon {
+      background-image: url(../images/logos/context/dotnet.svg);
```

Comment:
Oh, this isn't from platformicons?

Reply:
I see: https://github.com/getsentry/sentry/tree/master/src/sentry/static/sentry/images/logos/context

Should I copy the file here? 
I want to add this: https://github.com/getsentry/platformicons/blob/master/svg/dotnet.svg

Reply:
That's right - I pushed it there.
It's gonna look like this:

![image](https://user-images.githubusercontent.com/9060071/107484212-689d6c80-6b82-11eb-938b-0ebee45465f9.png)

If you prefer it to be rounded like this:

![image](https://user-images.githubusercontent.com/9060071/107484437-b31ee900-6b82-11eb-9736-01f8e699df5c.png)
You might wanna add:
```suggestion
      background-image: url(../images/logos/context/dotnet.svg);
      border-radius: 50%;
```

Reply:
In the future we might wanna refactor these context logos to use the ones directly in the platformicons repo - some of them are now duplicated in both places.